packages/enzyme-adapter-react-16.3/src/ReactSixteenThreeAdapter.js

```diff
@@ -417,27 +417,30 @@ class ReactSixteenThreeAdapter extends EnzymeAdapter {
   displayNameOfNode(node) {
     if (!node) return null;
     const { type, $$typeof } = node;
+    
+    const nodeType = type || $$typeof;
 
-    switch (type || $$typeof) {
-      case AsyncMode: return 'AsyncMode';
-      case Fragment: return 'Fragment';
-      case StrictMode: return 'StrictMode';
-      case Profiler: return 'Profiler';
```

Comment:
i wonder if a simpler change, since this is the 16.3 adapter, would be to just delete the Profiler line?